content/code-of-conduct.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Code of Conduct
+description: Kubernetes Code of Conduct
+---
+
+As contributors and maintainers of this project, and in the interest of fostering
```

Comment:
We can't have a copy pasted version. Can we programatically import it instead?

Reply:
@mrbobbytables I see we are importing the Community Guide using the hack/gen script. But since the Following CoC is not in the Kubernetes Root Repo but rather present in CNCF repository. What should be the approach ?

Reply:
The script can be updated to pull from repos outside of kubernetes, but it'll probably be sometime later this week before I can get to it.